re2 not compiling

Hey, I'm not longer able to compile cqerl. Looks like one if it dependencies (https://github.com/matehat/re2) is the root cause of the problem. I tried to replace it with the latest version from here https://github.com/tuncer/re2 and it worked fine.

```
...
===> Compiling setup
===> Compiling idna
===> Compiling bear
===> Package <<"pc">> not found. Fetching registry updates and trying again...
===> Updating package registry...
===> Writing registry to /root/.cache/rebar3/hex/default/registry
===> Generating package index...
===> [cloudi_service_oauth1:1.5.1] Only existing version of cloudi_service_db_riak is 1.3.3 which does not match constraint ~> 1.5.1. Using anyway, but it is not guaranteed to work.
===> Writing index to /root/.cache/rebar3/hex/default/packages.idx
===> Fetching pc ({pkg,<<"pc">>,<<"1.4.0">>})
===> Downloaded package, caching at /root/.cache/rebar3/hex/default/packages/pc-1.4.0.tar
===> Compiling pc
===> Compiling re2
===> Compiling /code/deps/re2/c_src/re2_nif.cpp
===> /code/deps/re2/c_src/re2_nif.cpp:8:21: fatal error: re2/re2.h: No such file or directory
 #include <re2/re2.h>
                     ^
compilation terminated.

** (Mix) Could not compile dependency :re2, "/root/.mix/rebar3 bare compile --paths "/code/_build/prod/lib/*/ebin"" command failed. You can recompile this dependency with "mix deps.compile re2", update it with "mix deps.update re2" or clean it with "mix deps.clean re2"
```

--- a/rebar.config
+++ b/rebar.config
@@ -6,5 +6,5 @@
   {semver, ".*", {git, "https://github.com/nebularis/semver.git", "c7d509"}},
   {uuid, ".*", {git, "https://github.com/okeuday/uuid.git", {tag, "v1.4.1"}}},
   {pooler, ".*", {git, "https://github.com/seth/pooler.git", {tag, "1.5.0"}}},
-  {re2, ".*", {git, "https://github.com/matehat/re2.git"}}
+  {re2, ".*", {git, "https://github.com/tuncer/re2.git", {tag, "v1.7.1"}}}
 ]}.

